### PR TITLE
Fix handling of Immunization notGiven and status

### PR DESCRIPTION
**Pull request by dotasek**
Fixes https://github.com/hapifhir/org.hl7.fhir.core/issues/1008

### CODE REVIEW

File: org.hl7.fhir.convertors/src/main/java/org/hl7/fhir/convertors/conv30_40/resources30_40/Immunization30_40.java [hunk 1]
```diff
@@ -10,10 +10,17 @@
...
+
 import org.hl7.fhir.exceptions.FHIRException;
+import org.hl7.fhir.r4.model.Extension;
+import org.hl7.fhir.r4.model.Immunization;
+
+
 
 public class Immunization30_40 {
 
+  public static final String NOT_GIVEN_EXTENSION_URL = "http://hl7.org/fhir/3.0/StructureDefinition/extension-Immunization.notGiven";
```
Comment (lukedegruchy) [inline org.hl7.fhir.convertors/src/main/java/org/hl7/fhir/convertors/conv30_40/resources30_40/Immunization30_40.java:22]: FYI:  Opening this URL gives me a page with

image

**Review by lukedegruchy — APPROVED** [review]

File: org.hl7.fhir.convertors/src/main/java/org/hl7/fhir/convertors/conv30_40/resources30_40/Immunization30_40.java (hunk 1 above)
Comment (dotasek) [reply org.hl7.fhir.convertors/src/main/java/org/hl7/fhir/convertors/conv30_40/resources30_40/Immunization30_40.java:22]: Those URLs don't need to resolve (ideally, the do, but they are not required).

It should match this from the R3->R4 conversion map:

```
src.notGiven as v ->  tgt.extension as vt,  vt.url = 'http://hl7.org/fhir/3.0/StructureDefinition/extension-Immunization.notGiven',  vt.value = v;
```

**Comment by dotasek** [comment]
From a conversion with @grahamegrieve: we should remove the extension on the way back from R4 to R3. There is a mechanism for doing this built in, so it shouldn't be a chunk of custom code.

**Review by grahamegrieve — APPROVED** [review]
ok

**Pull request merged**